packages/near-fast-auth-signer/.gitignore

```diff
@@ -25,3 +25,6 @@ yarn-error.log*
 
```

Comment:
<img width="631" alt="Screenshot 2024-01-25 at 3 01 09 PM" src="https://github.com/near/fast-auth-signer/assets/22358004/23807f74-b067-442f-8662-2a93e55b7f72">

On iPad mini the iFrame it's not properly rendered